images.json

```diff
@@ -37,10 +37,10 @@
       },
       {
         "name": "friendbot",
-        "repo": "stellar/go",
-        "ref": "horizon-v24.0.0",
+        "repo": "stellar/friendbot",
+        "ref": "d075991bca9365a9c87ab39f891dd3583e78c5c6",
```

Comment:
TODO:
- [ ] Update the commits in images.json for friendbot to the commit on the main branch of stellar/friendbot after these two changes are merged:
   - https://github.com/stellar/friendbot/pull/35
   - https://github.com/stellar/friendbot/pull/32